src/components/lib/cos_kernel_api.c

```diff
@@ -298,7 +299,7 @@ __page_bump_valloc(struct cos_compinfo *ci)
 
 		/* PTE */
 		if (call_cap_op(meta->captbl_cap, CAPTBL_OP_PGTBLACTIVATE,
-				pte_cap, meta->pgtbl_cap, ptemem_cap, 1)) {
+				pte_cap, meta->local_pgtbl_cap, ptemem_cap, 1)) {
```

Comment:
If we keep the local as `pgtbl_cap`, then we don't have all of these changes.  Shows that keeping local as default is likely useful.